module/VuFindApi/src/VuFindApi/Controller/ApiTrait.php

```diff
@@ -202,4 +224,42 @@ protected function output($data, $status, $httpCode = null, $message = '')
         $headers->addHeader($contentTypeHeader);
         return $response;
     }
+
+    /**
+     * Init API key settings
+     *
+     * @param array $settings API key settings from config.ini
+     *
+     * @return void;
+     */
+    protected function initApiKeySettings(array $settings): void
+    {
+        $this->developerSettingsService = $this->getService(DeveloperSettingsService::class);
+        $this->apiKeyHeaderField = $settings['header_field'] ?? 'X-API-KEY';
```

Comment:
Would it make sense to set the default in the property definition, and then override it like this?

```suggestion
        $this->apiKeyHeaderField = $settings['header_field'] ?? $this->apiKeyHeaderField;
```

That would make it easier to change the default on a per-controller basis without having to override this method.